gapic/templates/%namespace/%name_%version/%sub/services/%service/_shared_macros.j2

```diff
@@ -447,6 +448,17 @@ class _{{ name }}(_Base{{ service.name }}RestTransport._Base{{name}}, {{ async_m
         resp = {{ sig.response_type }}()
         resp = json_format.Parse(content, resp)
         resp = {{ await_prefix }}self._interceptor.post_{{ name|snake_case }}(resp)
+        if CLIENT_LOGGING_SUPPORTED and _LOGGER.isEnabledFor(logging.DEBUG):  # pragma: NO COVER
+            _LOGGER.debug(
+                "Received response for {{ service.meta.address.proto_package_versioned }}.{{ service.async_client_name }}.{{ name }}",
+                extra = {
+                    "serviceName": "{{ service.meta.address.proto }}",
+                    "rpcName": "{{ name }}",
+                    "retryAttempt": 1,
```

Comment:
should this be `"retryAttempt": 0`?

Reply:
I intended for `retryAttempt` to only be in the request log. We can remove it here.

Also `retryAttempt` with value 0 sounds good for the initial request

Reply:
We shouldn't have `retryAttempt` here at all, because this doesn't handle the retries and this value does not get incremented. In the test log output Omair posted, all the retries say `"retryAttempt": 1`, which is wrong.

TAL @ [go/cloudsdk-python-logging-locations](http://goto.google.com/cloudsdk-python-logging-locations) for a simple idea for addressing this.

Reply:
Resolving, as we decided to remove `retryAttempt` from this log event.